Formula/prql-compiler.rb

```diff
@@ -1,19 +1,17 @@
 class PrqlCompiler < Formula
```

Comment:
Do you think we could change the name of this file to `prqlc.rb`? Ideally the recent change should mean we use `prql/prql/prqlc` rather than `prql/prql/prql-compiler` (and I don't know whether it's possible to deprecate the old one or we just remove it, fine to remove if it that's easier...)

Reply:
It's entirely possible to both change the canonical formula name and keep the old name around as an alias!  I can make that change in a few hours when I get home.

Reply:
Super! Thank you

Reply:
Sorry about so long a delay!